doc/gitian-building.md

```diff
@@ -394,5 +396,5 @@ Uploading signatures
 ---------------------
 
 After building and signing you can push your signatures (both the `.assert` and `.assert.sig` files) to the
-[bitcoin/gitian.sigs](https://github.com/bitcoin/gitian.sigs/) repository, or if that's not possible create a pull
-request. You can also mail the files to me (laanwj@gmail.com) and I'll commit them.
+[dash/gitian.sigs](https://github.com/dashpay/gitian.sigs/) repository, or if that's not possible create a pull
```

Comment:
We once started storing the gitian.sigs in a Darkcoin repo, but never made to the point to use them to sign/verify the builds. I think it will be part of the improved release process